tests/unit/lib/test_signal_schema.py

```diff
@@ -2,8 +2,9 @@
 from typing import Optional, Union
 
 import pytest
+from sqlalchemy import Column
```

Comment:
Please use DataChain's Column, not sqlite. There is a chance we will be replacing the implementation.

Reply:
I had some strange issues when I was using `Column` from `DataChain` [here](https://github.com/iterative/datachain/pull/248/files#diff-69a1fc2d77ff106392ad7649ed8cf7ec344dad230af2e4d0ea9c64b816cee15bR236) so I change to use `sa.Column` directly. For some reason `index` field was defined when using `DataChain.Column` for each column and that was causing failure later on when creating table with those columns. 
I need to investigate why is that (maybe we accidentally broke something when defining our `Column` class which extends SQLAlchemy one) but regardless, this is all hidden from user, i.e is not in the API itself so it should be good.